sdk/schemaregistry/schema-registry/src/constants.ts

```diff
@@ -1,6 +1,4 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export const SDK_VERSION: string = "1.0.0-beta.2";
```

Comment:
remove the corresponding line in package.json for `constantPaths`

Reply:
Good catch! done.